app/templates/components/as-calendar.hbs

```diff
@@ -1,5 +1,8 @@
 {{#if showHeader}}
-  {{as-calendar/header title=title model=model}}
+  {{as-calendar/header 
+    title=title 
+    model=model 
+    onNavigateWeek=(action "onNavigateWeek")}}
```

Comment:
Doing that would mean that users of the component would be forced to define that attribute in their template when the use the calendar component. That's not good from a backward compatibility standpoint. As soon everyone updates to the latest release, they'll get an error like this:

```
Error: An action could not be made for `attrs.onNavigateWeek` in <dummy@component:as-calendar::ember384>. Please confirm that you are using either a quoted action name (i.e. `(action 'attrs.onNavigateWeek')`) or a function available in <dummy@component:as-calendar::ember384>.
```